external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -29,45 +30,36 @@ if(${ENABLE_PCMSolver})
                        -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}
                        -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_BINDIR=${CMAKE_INSTALL_BINDIR}
+                       -DPYMOD_INSTALL_LIBDIR=${PYMOD_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_DATADIR=${CMAKE_INSTALL_DATADIR}
                        -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
+                       -DENABLE_psi4=ON
                        -DSTATIC_LIBRARY_ONLY=${_a_only}
+                       -DSHARED_LIBRARY_ONLY=${_so_only}
 #                       -DENABLE_OPENMP=???
                        #-DENABLE_XHOST=???  # option missing
                        #-DENABLE_XHOST=${ENABLE_XHOST}  # option missing
                        #-DBUILD_FPIC=${BUILD_FPIC}  # always fpic'd
                        -DENABLE_GENERIC=${ENABLE_GENERIC}
-
-
-#                  -DENABLE_CXX11_SUPPORT=ON
-#                  -DENABLE_64BIT_INTEGERS=${ENABLE_64BIT_INTEGERS}
+                       -DENABLE_CXX11_SUPPORT=ON
+                       -DENABLE_64BIT_INTEGERS=ON
 #                  -DENABLE_TESTS=OFF
 #                  -DENABLE_LOGGER=OFF
 #                  -DENABLE_TIMER=OFF
 #                  -DBUILD_STANDALONE=OFF
 #                  -DENABLE_FORTRAN_API=OFF
 #                  -DSTATIC_LIBRARY_ONLY=${ENABLE_STATIC}
 #                  -DPYTHON_INTERPRETER=${PYTHON_EXECUTABLE}
```

Comment:
I only need the `PYTHON_INTERPRETER` to configure some of the '*.py` files, nothing more.
I had considered FindPythonLibsNew, but FindPythonInterp already does a good job at just finding the interpreter.